src/content/docs/integrations/kubernetes-integration/understand-use-data/auto-telemetry-pixie-data-model.mdx

```diff
@@ -0,0 +1,437 @@
+---
+title: Query Auto-telemetry with Pixie data
+tags:
+  - Pixie Auto-telemetry
+  - Service monitoring
+  - Kubernetes
+  - eBPF
+metaDescription:
+watermark: BETA 
+---
+
+<Callout variant="important">
+Auto-telemetry with Pixie is a supported public beta. Therefore, the features and functionality are subject to change. By joining the beta for Auto-telemetry with Pixie, you agree to the terms in the [New Relic Pre-Release Policy](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy/). 
+</Callout>
+
+Auto-telemetry with Pixie pulls data from the Pixie Cloud API and sends it to the New Relic OpenTelemetry endpoint. You can build your own charts and [query](/docs/using-new-relic/data/understand-data/query-new-relic-data) your Auto-telemetry with Pixie data using the query builder and the NerdGraph API.
+
+Find out more about getting started with Auto-telemetry with Pixie [here](/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie).
+
+## Metrics and specifications
+
+### HTTP metrics
```

Comment:
Maybe consider making these second level headers so people can find them in the right nav